### DISCUSSION TITLE
iPega 9087 S HELP

### DISCUSSION THREAD

**Discussion by dipip** [ndeadly/MissionControl#451] [Q&A]
I just installed the latest version and I still can't connect it to the console. It won't detect it. I've tried to pair it several times but still no luck. Why is this happening? I don't understand, I've done all the steps by the book and nothing. Help please.

**Comment by ndeadly** [comment]
If you've done everything correctly as per the instructions, the 9087s might be a Bluetooth LE controller. I don't currently support those. If you pair it with a PC you should be able to check the device properties to confirm.

**Reply by dipip** [reply]
Thank you so much for replying. I'll look into it.
